src/main/java/com/metriql/warehouse/bigquery/BigQueryWarehouse.kt

```diff
@@ -33,7 +33,7 @@ object BigQueryWarehouse : Warehouse<BigQueryWarehouse.BigQueryConfig> {
         val location: String? = null,
         val priority: String? = null,
         val retries: Int? = null,
-        val keyFile: String? = null,
+        val keyfile: String? = null,
```

Comment:
Can you please add `@JsonAlias('key_file')` here so that we make it backward compatible?

Reply:
Sure. I added it.